gulpfile.js

```diff
@@ -438,7 +438,7 @@ function createSandboxExternal(defines) {
     transform("utf8", content => {
       content = preprocessor2.preprocessPDFJSCode(ctx, content);
       return `${licenseHeader}\n${content}`;
-    })
+    }).pipe(rename("pdf.sandbox.external.sys.mjs"))
```

Comment:
If you instead rename the file itself to `pdf.sandbox.external.sys.mjs` this line shouldn't be necessary.

Reply:
I tried that but it doesn't work because of:
https://github.com/mozilla/pdf.js/blob/384bd961657a444b9ec3cb05042ee5f220565122/src/pdf.sandbox.js#L17